pkg/commands/git_commands/repo_paths.go

```diff
@@ -228,7 +228,7 @@ func getCurrentRepoGitDirPath(
 }
 
 // takes a path containing a symlink and returns the true path
-func resolveSymlink(path string) (string, error) {
+func ResolveSymlink(path string) (string, error) {
 	l, err := os.Lstat(path)
```

Comment:
Why do we even need this check if it's `os.ModeSymlink`?

In order to be able to call `lazygit` from a folder whose *path* _contains_ a symlink this check fails, and `lazygit` fails to start, but if we just `return filepath.EvalSymlinks(path)`, all works as expected.

The example is what I've written in the conversation for this PR.

@jesseduffield could this have been done for potential performance reasons? Tried blaming it, but it went through two refactors so I stopped.

Reply:
I've never made any performance-related decisions around symlinks. If `filepath.EvalSymlinks(path)` works I'm happy for us to use it

Reply:
The docs say:
```
// EvalSymlinks returns the path name after the evaluation of any symbolic
// links.
// If path is relative the result will be relative to the current directory,
// unless one of the components is an absolute symbolic link.
// EvalSymlinks calls Clean on the result.
```

That would suggest nothing changes if the given path _isn't_ a symlink, so I see no reason to `if-else` on it being a `symlink`.

But I'd wait for tests that confirm it.